Tests/UIKitTests/UITableViewExtensionsTests.swift

```diff
@@ -53,10 +53,8 @@ final class UITableViewExtensionsTests: XCTestCase {
 	}
 	
 	func testRemoveTableFooterView() {
-		tableView.tableFooterView = UIView()
-		XCTAssertNotNil(tableView.tableFooterView)
 		tableView.removeTableFooterView()
-		XCTAssertNil(tableView.tableFooterView)
+		XCTAssertEqual((tableView.tableFooterView ?? UIView()).frame.height, 0)
```

Comment:
I would modify this test because we don't want to default back to another view. Better to check for nil explicitly and then compare frames